### ISSUE TITLE
Document Offline / Desktop / File based MEW

### ISSUE THREAD

**Issue by jidanni** [MyEtherWallet/MyEtherWallet#934]
Let's say somebody does
```
$ wget https://github.com/MyEtherWallet/MyEtherWallet/archive/0.1.0.tar.gz
$ tar xzvf 0.1.0.tar.gz
$ $BROWSER MyEtherWallet-0.1.0/common/index.html
```
Well in this case it would be good for him to see
```
Components not fully installed!
Please do ... first.
Then ... .
```
instead of just a blank screen.

**Comment by dternyak** [comment]
Thanks for the note @jidanni 

I'll use this issue as a center for speccing out what additional documentation we should have published before moving into a public release

- [ ] Inform users that `:downloadable` should be used if running MEW via `file` is desirable.
- [ ] Document / ensure Electron build docs are digest-able and visible

Please feel free to add to this list so we can have a comprehensive source of what additional documentation is critical to have before our production launch.

**Comment by jidanni** [comment]
Thanks. (By the way, common/index.html is already packed with many javascript and even ```<noscript>``` tests to detect if it is being run wrong. Just this fall through case was not detected. In any case do be sure a blank screen is never a possibility.)

**Comment by wbobeirne** [comment]
Addressed with new readme in #954.

To your point about index.html, `<noscript>` does not work for that because your browser still has Javascript enabled, so noscript tags don't show. Given that the readme explains how to run this, and downloading the repo is meant only for developers, I'm going to leave this as-is.
